protobuf/flow/access/access.proto

```diff
@@ -424,6 +446,42 @@ message GetAccountAtBlockHeightRequest {
   uint64 block_height = 2;
 }
 
+message GetAccountBalanceRequest {
+  bytes address = 1;
+}
+
+message GetAccountBalanceAtLatestBlockRequest {
+  bytes address = 1;
+}
+
+message GetAccountBalanceAtBlockHeightRequest {
+  bytes address = 1;
+  uint64 block_height = 2;
+}
+
+message AccountBalanceResponse {
+  uint64 account_balance = 1;
+  entities.Metadata metadata = 2;
+}
+
+message GetAccountKeysRequest {
+  bytes address = 1;
+}
+
+message GetAccountKeysAtLatestBlockRequest {
+  bytes address = 1;
+}
```

Comment:
Let's also provide support for specifying a specific key index, which is something currently supported by the REST API. since this field will be a number whose default (0) is a valid index, we'll need to wrap it in another message to allow the server to differentiate 0 and unset.

something like
```suggestion
message GetAccountKeysAtLatestBlockRequest {
  // address of account 
  bytes address = 1;
  // index of key to return
  // omit to return all keys
  entities.AccountKeyIndex index = 2;
}
```

then you could define this in https://github.com/onflow/flow/blob/master/protobuf/flow/entities/account.proto
```golang
message AccountKeyIndex {
  uint32 value = 1;
}
```